### ISSUE TITLE
cursorWordStartLeft regression makes VS compatibility impossible

### ISSUE THREAD

**Issue by zeromus** [Microsoft/vscode#51119]
- VSCode Version: 1.23.1
- OS Version: Windows 7
Does this issue occur when all extensions are disabled?: Yes

We've been repeatedly advised to use cursorWordStartRight to get VS-like behaviour. This depends on leaving ctrl+left bound to cusorWordStartLeft.

With https://github.com/Microsoft/vscode/commit/a811707 the behaviour of cursorWordStartLeft was changed. Consequently, it's not possible to mimic visual studio. I checked in 1.22.2 before that commit landed and it was working like visual studio.

Consider this test in 1.23.1:

this.is.a.test

Using cursorWordStartRight we get 7 steps, as we do in visual studio. 
Using cursorWordStartLeft we get 4 steps. 

What? Are there 7 word starts or 4? Seems crazy. Well, that's your business. I need it to be 7.

Please add an option to control it. I suggest something like `editor.wordSelectionWindowsStyle`. Here's some analysis. It works this way on:

Visual studio, wordpad, EmEditor, tortoisemerge, skype, mirc, Internet Explorer (except for the URL bar), openoffice writer (in windows, at least), keepass, the textboxes in the dropbox windows client, textboxes in oracle VirtualBox, wireshark's startup capture filter entry textbox.

Basically everything on windows that's not a web browser or an ancient editbox.
